cmd/provisioner-localpv/app/provisioner.go

```diff
@@ -187,29 +132,16 @@ func (p *Provisioner) Delete(pv *v1.PersistentVolume) (err error) {
 		err = errors.Wrapf(err, "failed to delete volume %v", pv.Name)
 	}()
 
-	//Determine the path and node of the Local PV.
-	path, node, err := p.getPathAndNodeForPV(pv)
-	if err != nil {
-		return err
-	}
-
 	//Initiate clean up only when reclaim policy is not retain.
 	if pv.Spec.PersistentVolumeReclaimPolicy != v1.PersistentVolumeReclaimRetain {
-		glog.Infof("Deleting volume %v at %v:%v", pv.Name, node, path)
-		cleanupCmdsForPath := []string{"rm", "-rf"}
-		podOpts := &HelperPodOptions{
-			cmdsForPath: cleanupCmdsForPath,
-			name:        pv.Name,
-			path:        path,
-			nodeName:    node,
+		//TODO: Determine the type of PV
+		pvType := GetLocalPVType(pv)
+		if pvType == "local-device" {
```

Comment:
can we create constants for hardcoded strings and also put a comment about "local-device", "hostpath" and "device"

Reply:
Will be doing as part of code-cleanup.